RTP implementation (#1)
awslabs/amazon-kinesis-video-streams-rtp: README.md

        |                           Timestamp                           |
        +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
        |           Synchronization Source (SSRC) identifier            |
        +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
        |       Contributing Source (CSRC[0..15]) identifiers           |
        |                             ....                              |
        +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+

        RTP Header extension:

         0                   1                   2                   3
         0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
        +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
        |            Profile            |           Length              |
        +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
        |                        Header Extension                       |
        |                             ....                              |
```

## Using the library

### Serializer
    1. Call `Rtp_Init()` to initialize the RTP Context.

>     2. Populate the fields of the RTP packet with the desired values.

Reviewer: ```suggestion
    2. Initialize an RtpPacket_t with the desired values.
```

>     3. Call `Rtp_Serialize()` to serialize the RTP packet passed.

Reviewer: ```suggestion
    3. Call `Rtp_Serialize()` to serialize the RtpPacket_t  passed.
```

### Deserializer
    1. Call `Rtp_Init()` to initialize the RTP Context.
    2. Pass the serialized packet along with its length to `Rtp_DeSerialize()` to
       deserialize the packet.

## Packetization
    1. Call `<Codec>Packetization_Init()` to intitializae the particular codec context.
    2. In case of H.264 Codedc packetization,` is returned. call `H264Packetizer_AddFrame()` or
       `H264Packetizer_AddNalu()` to add frame or individual NALUs.
    3. Call `<Codec>Packetizer_GetPacket()` in a loop to retrieve the packets iteratively

>        unless `<Codec>_RESULT_NO_MORE_PACKETS` is returned.

Reviewer: ```suggestion
    2. In case of H.264 Codec packetization - 
        - Call `H264Packetizer_AddFrame()` to add  a frame OR
        - Call `H264Packetizer_AddNalu()` repeatedly to add NALUs of a frame.
    3. For all codes  other than H.264, call `<Codec>Packetizer_AddFrame()` to add  a frame.
    4. Call `<Codec>Packetizer_GetPacket()` repeatedly to retrieve the packets 
       until `<Codec>_RESULT_NO_MORE_PACKETS` is returned.
```
